agents/windows/plugins/win_license.bat

```diff
@@ -5,4 +5,13 @@ REM * plugin to gather and output Windows activation status
 REM ***
 
 echo ^<^<^<win_license^>^>^>
-cscript //NoLogo %windir%/System32/slmgr.vbs /dli
+if not exist "%temp%\slmgr.vbs" (
+    copy "%windir%\system32\slmgr.vbs" "%temp%\slmgr.vbs" > NUL
+)
+
+if exist "%temp%\slmgr.vbs" (
```

Comment:
A comment about the localization behavior here would be good. I know it will also be in the git log, but a comment is immediately visible and this behavior is not intuitive.